jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -1227,14 +1220,14 @@ public void failed(Throwable x)
         {
             if (LOG.isDebugEnabled())
                 LOG.debug("write failed {}", this, x);
-            // Called when an individual write succeeds.
```

Comment:
Ditto on comment removal?

Reply:
Because it's wrong (should be "write fails") and repeats exactly what's written in the log the line above.